app/src/migration/java/org/mozilla/fenix/MigratingFenixApplication.kt

```diff
@@ -29,11 +29,20 @@ class MigratingFenixApplication : FenixApplication() {
 
     val migrationStore by lazy { MigrationStore() }
 
+    val migrationPushSubscriber by lazy {
+        MigrationPushSubscriber(
+            this,
+            components.backgroundServices.pushService,
```

Comment:
`pushService` is `private`, so this won't compile.

Reply:
This was in error from `git add`.